src/main/java/org/jmhsrobotics/frc2025/RobotContainer.java

```diff
@@ -53,7 +53,7 @@ public class RobotContainer {
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
     // Change to "SingleControl" or "DoubleControl" here based on preference
-    this.control = new SingleControl();
+    this.control = new AltControlMode();
 
```

Comment:
Looks good, only comment is we may want to leave the robot in single control. Also we may want to look at adding a way to switch control modes without code changes. This will be challenging because we are using triggers.